From 8a0de524f079651a6672907ef665679d32f547e2 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Fri, 17 Aug 2018 12:38:06 -0700
Subject: [PATCH 1/5] lineage: Enable QCOM sepolicy

 * Also includes the repo for legacy devices (those that never got O)

Change-Id: I6cddcece3b7e827fd296a84d14122396aad5fd77
---
 snippets/lineage.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 27728fd..153e868 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -53,7 +53,8 @@
   <project path="vendor/lineage" name="LineageOS/android_vendor_lineage" />
 
   <!-- CodeAurora additions -->
-  <!--<project path="device/qcom/sepolicy" name="LineageOS/android_device_qcom_sepolicy" />-->
+  <project path="device/qcom/sepolicy" name="LineageOS/android_device_qcom_sepolicy" />
+  <project path="device/qcom/sepolicy-legacy" name="LineageOS/android_device_qcom_sepolicy-legacy" />
   <project path="external/ant-wireless/ant_native" name="LineageOS/android_external_ant-wireless_ant_native" />
   <project path="external/ant-wireless/ant_service" name="LineageOS/android_external_ant-wireless_ant_service" />
   <project path="external/ant-wireless/antradio-library" name="LineageOS/android_external_ant-wireless_antradio-library" />
-- 
2.17.1

